specs/networking/p2p-interface.md

```diff
@@ -177,19 +196,22 @@ Clients MUST reject (fail validation) messages that are over this size limit. Li
 The payload is carried in the `data` field of a gossipsub message, and varies depending on the topic:
 
 
-| Topic                        | Message Type      |
-|------------------------------|-------------------|
-| beacon_block                 | BeaconBlock       |
-| beacon_attestation           | Attestation       |
-| shard{N}\_beacon_attestation | Attestation       |
-| voluntary_exit               | VoluntaryExit     |
-| proposer_slashing            | ProposerSlashing  |
-| attester_slashing            | AttesterSlashing  |
+| Topic                                  | Message Type      |
+|----------------------------------------|-------------------|
+| beacon_block                           | BeaconBlock       |
+| beacon_aggregate_and_proof             | AggregateAndProof |
+| beacon_attestation\*                   | Attestation       |
+| committee_index{N}\_beacon_attestation | Attestation       |
+| voluntary_exit                         | VoluntaryExit     |
+| proposer_slashing                      | ProposerSlashing  |
+| attester_slashing                      | AttesterSlashing  |
```

Comment:
IMO the current "Topics" and "Messages" sections are difficult to look up.
It would be nice to (i) have a single table to explain the topics and (ii) move the `beacon_aggregate_and_proof` and `committee_index{N}\_beacon_attestation` validation logic explainer to its own section.

Reply:
did a bit of a reorg. Think it is better now